blocks/library/heading/index.js

```diff
@@ -29,6 +29,8 @@ registerBlockType( 'core/heading', {
 
 	className: false,
 
+	supportAnchor: true,
```

Comment:
There was another related discussion previously, I think when `className` was introduced, about the proliferation/scalability of properties on the block API. It may have been @BE-Webdesign that raised it? In any case, it seems wise to build a pattern around such "boolean" features (either block supports it or it doesn't). Even the prefix as it stands here is indicative of scaling trouble: `supportAnchor`, eventually we might need `supportFoo`, `supportBar`.

I don't mean for it to cause trouble for this pull request specifically, but if it's something we feel we ought to address, I think it should either occur here or prior to this pull request being merged.